src/services/users/index.js

```diff
@@ -60,7 +58,7 @@ const signInUserByEmail = async (email, password) => {
 	const refreshToken = await generateRefreshToken()
 	await updateRefreshToken(refreshToken , user.id)
 
-	return { accessToken, refreshToken, userId: user.id }
+	return { accessToken, refreshToken, userId: user.id,user }
```

Comment:
the refresh token and user id are in the user @beblicarl, they were separated at first because we were not sending all the info about the user, but we are now

```suggestion
	return { accessToken, user }
```